src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -926,6 +926,33 @@ public void deleteMetadata(String templateName, String scope) {
                 upload(this.getAPI(), this.getID(), inputStream, url, fileName, fileSize);
     }
 
+    /**
+     * Creates a new Metadata Cascade Policy on a folder.
+     *
+     * @param scope         the scope of the metadata cascade policy.
+     * @param templateKey   the key of the template.
+     * @return  information about the Metadata Cascade Policy.
+     */
+    public BoxMetadataCascadePolicy.Info createCascadePolicy(String scope, String templateKey) {
+        BoxMetadataCascadePolicy.Info policyInfo = BoxMetadataCascadePolicy
+                .create(this.getAPI(), this.getID(), scope, templateKey);
+
+        return policyInfo;
+    }
+
+    /**
+     * Retrieves all Metadata Cascade Policies on a folder.
+     *
+     * @param fields            optional fields to retrieve for cascade policies.
+     * @return  the Iterable of Box Metadata Cascade Policies in your enterprise.
+     */
+    public Iterable<BoxMetadataCascadePolicy.Info> getCascadePolicies(String... fields) {
```

Comment:
We should have an overload of this that allows passing in the owner enterprise ID